### ISSUE TITLE
Projects don't save map offset

### ISSUE THREAD

**Issue by karliss** [rizinorg/rizin#3967]
### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | ArchLinux x68_64
| File format of the file you reverse (mandatory)      | raw
| Architecture/bits of the file (mandatory)            | arm thumb (probably affects most)
| `rizin -v` full output, **not truncated** (mandatory)  | rizin 0.7.0 @ linux-x86-64 commit: 444000416cc929d4ce67116a46eaeee1d8a5b4fd

### Expected behavior

After loading project raw binary is mapped at the same address it was when saving.

This is important when working with raw MCU firmware binaries. This already annoying while working with single file, and would be even worse if i wanted to map multiple files for main firmware, bootloader and builtin bootloader.

### Actual behavior

Binary is mapped at 0

### Steps to reproduce the behavior
- Any plaintext file will do doesn't have to even be a real binary (don't use ELF/PE as rizin will interpret those differently)
-  https://asciinema.org/a/74deu2Bm5rcnpPgaPODoeDkaC
- `rizin -a arm -b 16 -m 0x2400 ./testfile`
- `oml` inspect the mapping table, can be also observed by seeking to corresponding position and printing assembly
- `Ps testproj.rzdb`
- `exit`
- `rizin`
- `Po testproj.rzdb`
- observe that file isn't mapped at 0x2400 anymore

**Comment by karliss** [comment]
I guess it's related to #713 . Were there any progress towards it?

**Comment by XVilka** [comment]
Not really, it wasn't touched since. Serialization of all of RzIO is big
task because of various inherently different modes. But it's possible to do
the basic subset serialization first, for the most common tasks.
